app/models/item_type.rb

```diff
@@ -6,13 +7,20 @@ class ItemType < ActiveRecord::Base
   serialize :allowed_keys, Array
   before_validation -> { self.allowed_keys = allowed_keys.map(&:to_sym) }
 
-  validates :name, presence: true
+  validates :name, :creator_id, presence: true
```

Comment:
Changed!
On Mar 21, 2016 4:51 PM, "David Faulkenberry" <notifications@github.com>
wrote:

> In app/models/item_type.rb
> <https://github.com/umts/aggressive-epsilon/pull/19#discussion_r56894704>:
>
> > @@ -6,13 +7,20 @@ class ItemType < ActiveRecord::Base
> >    serialize :allowed_keys, Array
> >    before_validation -> { self.allowed_keys = allowed_keys.map(&:to_sym) }
> >
> > -  validates :name, presence: true
> > +  validates :name, :creator_id, presence: true
>
> Better to validate the presence of the creator itself rather than its ID.
> It is possible for an item type to point to a creator service which doesn't
> actually exist.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/umts/aggressive-epsilon/pull/19/files/774f93452527087fd4154a31ad084677d4ac6c01#r56894704>
>